src/HasTranslations.php

```diff
@@ -140,6 +144,21 @@ public function forgetTranslation(string $key, string $locale): self
         return $this;
     }
 
+    public function forgetTranslations(string $key, bool $asNull = false): self
```

Comment:
Let's maybe ditch the `asNull` method. We'll add it if anybody really needs it.

Reply:
Actually, I found the asNull option useful - I had a case to set it to null in observer upon updating the row.

Reply:
I added the `asNull` option because I needed the option in one of my projects.